AirbnbSwiftFormatTool/AirbnbSwiftFormatTool.swift

```diff
@@ -0,0 +1,84 @@
+import ArgumentParser
+import Foundation
+
+/// A command line tool that formats the given directories using SwiftFormat and SwiftLint,
+/// based on the Airbnb Swift Style Guide
+@main
+struct AirbnbSwiftFormatTool: ParsableCommand {
+
+  // MARK: Internal
+
+  @Argument(help: "The directories to format")
+  var directories: [String]
+
+  @Option(help: "The abosolute path to a SwiftFormat binary")
+  var swiftFormatPath: String
+
+  @Option(help: "The path to use for SwiftFormat's cache")
+  var swiftFormatCachePath: String?
+
+  @Option(help: "The abosolute path to a SwiftLint binary")
+  var swiftLintPath: String
+
+  @Option(help: "The path to use for SwiftLint's cache")
+  var swiftLintCachePath: String?
+
+  @Flag(help: "When true, source files are not reformatted")
+  var lint = false
+
+  @Option(help: "The absolute path to the SwiftFormat config file")
+  var swiftFormatConfig = Bundle.module.path(forResource: "airbnb", ofType: "swiftformat")!
+
+  @Option(help: "The absolute path to the SwiftLint config file")
+  var swiftLintConfig = Bundle.module.path(forResource: "swiftlint", ofType: "yml")!
+
+  mutating func run() throws {
+    try swiftFormat.run()
+    swiftFormat.waitUntilExit()
+
+    try swiftLint.run()
+    swiftLint.waitUntilExit()
+  }
+
+  // MARK: Private
+
+  private lazy var swiftFormat: Process = {
+    var arguments = directories + [
+      "--config", swiftFormatConfig,
+    ]
+
+    if let swiftFormatCachePath = swiftFormatCachePath {
+      arguments += ["--cache", swiftFormatCachePath]
+    }
+
+    if lint {
+      arguments += ["--lint"]
+    }
+
+    let swiftFormat = Process()
+    swiftFormat.launchPath = swiftFormatPath
+    swiftFormat.arguments = arguments
+    return swiftFormat
+  }()
+
+  private lazy var swiftLint: Process = {
+    var arguments = directories + [
+      "--config", swiftLintConfig,
+      "--in-process-sourcekit",
```

Comment:
Lazy q: what does this do and why do we specify it?

Reply:
Will add a comment -- this is required due to SPM plugin sandboxing, I got this tip from https://theswiftdev.com/beginners-guide-to-swift-package-manager-command-plugins/